import-scripts/annotation_suite/annotation_suite_wrapper.sh

```diff
@@ -0,0 +1,128 @@
+    echo -e "\t-c | --center-name                   name of the center being processed"
+    echo -e "\t-s | --sequence-source               name of the sequence source used by the center (i.e., WXS, WGS)"
+    echo -e "\t-p | --annotation-suite-scripts-dir  path to the annotation suite scripts directory"
+}
+
+# parse input arguments
+for i in "$@"; do
+case $i in
+    -i=*|--input-directory=*)
+    INPUT_DATA_DIRECTORY="${i#*=}"
+    echo -e "\tINPUT_DATA_DIRECTORY=${INPUT_DATA_DIRECTORY}"
+    shift
+    ;;
+    -o=*|--output-directory=*)
+    OUTPUT_DATA_DIRECTORY="${i#*=}"
+    echo -e "\tOUTPUT_DATA_DIRECTORY=${OUTPUT_DATA_DIRECTORY}"
+    shift
+    ;;
+    -m=*|--merged-mutation-file=*)
+    MERGED_MUTATION_FILENAME="${i#*=}"
+    echo -e "\tMERGED_MUTATION_FILENAME=${MERGED_MUTATION_FILENAME}"
+    shift
+    ;;
+    -c=*|--center-name=*)
+    CENTER_NAME="${i#*=}"
+    echo -e "\tCENTER_NAME=${CENTER_NAME}"
+    shift
+    ;;
+    -s=*|--sequence-source=*)
+    SEQUENCE_SOURCE="${i#*=}"
+    echo -e "\tSEQUENCE_SOURCE=${SEQUENCE_SOURCE}"
+    shift
+    ;;
+    -p=*|--annotation-suite-scripts-dir=*)
+    ANNOTATION_SUITE_SCRIPTS_HOME="${i#*=}"
+    echo -e "\tANNOTATION_SUITE_SCRIPTS_HOME=${ANNOTATION_SUITE_SCRIPTS_HOME}"
+    shift
+    ;;
+    *)
+    ;;
+esac
+done
+
+if [[ -z ${INPUT_DATA_DIRECTORY} || -z ${OUTPUT_DATA_DIRECTORY} || -z ${MERGED_MUTATION_FILENAME} || -z ${CENTER_NAME} || -z ${SEQUENCE_SOURCE} || -z ${ANNOTATION_SUITE_SCRIPTS_HOME} ]]; then
+    usage
+    exit 1
+fi
+
+if [ -z ${JAVA_SSL_ARGS} ] ; then
+    echo "Cannot run the annotator without SSL args currently. exiting..."
+    exit 2
+fi
+
+PROCESSED_SUB_DIR_NAME="${OUTPUT_DATA_DIRECTORY}/processed"
+ANNOTATED_SUB_DIR_NAME="${OUTPUT_DATA_DIRECTORY}/annotated"
+FILE_EXTENSIONS_LIST="vcf,maf,txt" # text files are treated as MAFs to handle names like data_mutations_extended.txt
+
+STANDARDIZE_MUTATION_DATA_SCRIPT=${ANNOTATION_SUITE_SCRIPTS_HOME}/standardize_mutation_data.py
+GENOME_NEXUS_ANNOTATOR_JAR=${ANNOTATION_SUITE_SCRIPTS_HOME}/annotator.jar
+MERGE_MAFS_SCRIPT=${ANNOTATION_SUITE_SCRIPTS_HOME}/merge_mafs.py
+
+GENOME_NEXUS_ANNOTATOR_ISOFORM="uniprot"
+GENOME_NEXUS_ANNOTATOR_POST_SIZE=1000
+
+function initAndCleanWorkingDirectory {
+    # (1): parent directory containing raw/unprocessed data files
+    data_dir=$1
+    if [ -d $data_dir ] ; then
+        rm -f $data_dir/*
+    else
+        mkdir $data_dir
+    fi
+}
+
+function standardizeMutationFilesFromDirectory {
+    # processed files will be written to ${OUTPUT_DATA_DIRECTORY}/processed
+    echo -e "\t[INFO] standardizeMutationFilesFromDirectory(), standardized mutation files from ${INPUT_DATA_DIRECTORY} will be written to ${PROCESSED_SUB_DIR_NAME}"
+    python3 ${STANDARDIZE_MUTATION_DATA_SCRIPT} --input-directory ${INPUT_DATA_DIRECTORY} --output-directory ${PROCESSED_SUB_DIR_NAME} --center ${CENTER_NAME} --sequence-source ${SEQUENCE_SOURCE} --extensions ${FILE_EXTENSIONS_LIST}
```

Comment:
I dont think this will be necessary - if it doesnt exist then it wont be able to run and would exit anyway. Also given that the motivation for building this is for GENIE and Tom specifically runs python3 also makes me believe that this type of check isn't really necessary but I understand.

Reply:
Actually now that I recall we want to migrate all of our python scripts to python3 at some point also.